### PR TITLE
PG interface to cluster for acceptance tests

### PR DESCRIPTION
**Pull request by vivekmenezes**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4106)

### CODE REVIEW

**Comment by petermattis** [comment]
LGTM

---

Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---

*[acceptance/cluster/localcluster.go, line 606 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4106#-K9c-uhIB2hnTesfH8Xk:-K9c-uhIB2hnTesfH8Xl:-1477484594) ([raw file](https://github.com/cockroachdb/cockroach/blob/2b278035d44e9df6ef74341bc88014ef26bb3f68/acceptance/cluster/localcluster.go#L606)):*
`PGUrl`. This would match the naming of `testutils/squtils.PGUrl`.

---

*[acceptance/main_test.go, line 145 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4106#-K9c-ekcoGkwRvoslr6R:-K9c-ekcoGkwRvoslr6S:1180598308) ([raw file](https://github.com/cockroachdb/cockroach/blob/2b278035d44e9df6ef74341bc88014ef26bb3f68/acceptance/main_test.go#L145)):*
Placing this test in `main_test.go` seems awkward. And I'm not even sure it is necessary. As soon as we add an acceptance test which utilizes `makePGClient` and `PGConnString` we'll have coverage of this new functionality.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4106#-:-K9c0AqVgJ5pd0CdSNX_:676834233)*

**Comment by vivekmenezes** [comment]
Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---

*[acceptance/cluster/localcluster.go, line 606 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4106#-K9c-uhIB2hnTesfH8Xk:-K9c24bZa8iWroKDkQoB:677042770) ([raw file](https://github.com/cockroachdb/cockroach/blob/2b278035d44e9df6ef74341bc88014ef26bb3f68/acceptance/cluster/localcluster.go#L606)):*
SGTM

---

*[acceptance/main_test.go, line 145 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4106#-K9c-ekcoGkwRvoslr6R:-K9c1yZvZL2oGA3vrPZF:871922127) ([raw file](https://github.com/cockroachdb/cockroach/blob/2b278035d44e9df6ef74341bc88014ef26bb3f68/acceptance/main_test.go#L145)):*
true. I'll take it out when I add in the bank example.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4106)*

File: acceptance/cluster/localcluster.go [hunk 1]
```diff
@@ -601,6 +603,23 @@ func (l *LocalCluster) PGAddr(i int) *net.TCPAddr {
 	return l.Nodes[i].PGAddr()
 }
 
+// PGUrl returns a URL string for the given node postgres server.
+func (l *LocalCluster) PGUrl(i int) string {
+	certUser := security.RootUser
+	options := url.Values{}
```
Comment (tamird) [inline acceptance/cluster/localcluster.go]: This is nice. Can you give this treatment to `sqlutils.PGUrl` too?

Comment (vivekmenezes) [inline acceptance/cluster/localcluster.go]: ok

**Pull request merged**